app-size-analyzer.code-workspace

```diff
@@ -0,0 +1,129 @@
+{
```

Comment:
should we add this to the repo vs to the .gitignore? unclear to me, but fwiw it's ignored for the monorepo

Reply:
Hmm my understanding is this is supposed to be checked in, similar to Xcode's workspace file.